### ISSUE TITLE
Downloads

### ISSUE THREAD

**Issue by Johnthomas777** [bagaswastu/librebook#10]
Waiting on server reply error, I don't know if your aware of it

**Comment by bgwastu** [comment]
Hi, thanks for reporting. Have you tried using the [latest version](https://codemagic.io/app/6003d2461b08f3ec61b49785/build/608f81f929657a0a54e7b4bc) of the application?

**Comment by Johnthomas777** [comment]
Latest version 0.1.0 alpha???

**Comment by bgwastu** [comment]
Yes, you can download it in codemagic. If there are still problems, don't
hesitate to report again.

On Thu, May 27, 2021, 12:06 AM bagaswastu/librebook -
reply+aqfpflsf53efismjjcyfgmv6xjqrnevbnhhdllnevm at reply.github.com <

**Comment by Johnthomas777** [comment]
Yep that's the version I have, managed to download a few books just a bit
slower then it was, no buggy just thought I'd let u know, great app by the
way love it

**Comment by bgwastu** [comment]
Thank you, maybe later I'll try to check it out.

On Thu, May 27, 2021, 12:14 AM bagaswastu/librebook -
reply+aqfpflwwvyj32r5bi2yvpjv6xjrpdevbnhhdllnevm at reply.github.com <
